mappings/net/minecraft/client/gui/screen/CreditsScreen.mapping

```diff
@@ -4,15 +4,18 @@ CLASS net/minecraft/unmapped/C_emwfaans net/minecraft/client/gui/screen/CreditsS
 	FIELD f_cuxehlzw SEPARATOR_LINE Lnet/minecraft/unmapped/C_rdaqiwdt;
 	FIELD f_cxzuezzk CENTERED_LINE_PREFIX Ljava/lang/String;
 	FIELD f_dakivsbd logoRenderer Lnet/minecraft/unmapped/C_siglwpps;
+	FIELD f_ewbhrgtm POST_CREDITS_TEXT Lnet/minecraft/unmapped/C_ncpywfca;
 	FIELD f_ftsdfuxo centeredLines Lit/unimi/dsi/fastutil/ints/IntSet;
 	FIELD f_fyszbzgh speed F
+	FIELD f_hefuvaro END_POEM Lnet/minecraft/unmapped/C_ncpywfca;
```

Comment:
This name might be a bit too creative since it doesn't match the string `"texts/end.txt"`, but it feels more recognizable than `END_TEXT`

Reply:
mojmap is `END_POEM_LOCATION`, so I guess not too creative after all

Reply:
the current name is good